lisa/features/startstop.py

```diff
@@ -1,10 +1,19 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+from enum import Enum
 
 from lisa.feature import Feature
 
 FEATURE_NAME_STARTSTOP = "StartStop"
 
+DeallocateState = Enum(
```

Comment:
Use the pattern like below, it supports prompt on names. And not use "deallocate", it's an Azure name.

```python
class StopState(str, Enum):
    Hibernate = "hibernate "
    Shutdown= "shutdown"
```

Reply:
updated